producers-api/src-gen/main/java/uk/gov/moj/sdt/ws/_2013/sdt/baseschema/package-info.java

```diff
@@ -0,0 +1,2 @@
+@javax.xml.bind.annotation.XmlSchema(namespace = "http://ws.sdt.moj.gov.uk/2013/sdt/BaseSchema", elementFormDefault = javax.xml.bind.annotation.XmlNsForm.QUALIFIED)
+package uk.gov.moj.sdt.ws._2013.sdt.baseschema;
```

Comment:
Previously all of the classes under producers-api/src-gen were generated from the WSDL files under producers-api/src/resources using Apache CXF WSDL2Java.  If any changes needed to be made to them the WSDL was updated and the classes regenerated.  Were all the package-info classes generated from the WSDL?  If not they could be lost or become outdated if the WSDL changes and the classes need to be regenerated.

There probably needs to be a gradle task added to generate these classes.

Reply:
changes done, please review again

Reply:
Classes now generated from WSDL tasks in producers-api build.gradle.